gradio/components/highlighted_text.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         """
         Parameters:
             value: Default value to show. If callable, the function will be called whenever the app loads to set the initial value of the component.
+            color_map: A dictionary mapping labels to colors. The colors must be specified as hex codes.
```

Comment:
I don't think they must be specified as hex codes actually. Would be good to maybe give an example, e.g. `{"person": "red", "location": "#FFEE22"}